prog/vm/host_nexus.rb

```diff
@@ -117,11 +117,22 @@ def before_run
       spdk_installation = vm_host.spdk_installations.first
       spdk_cores = (spdk_installation.cpu_count * vm_host.total_cores) / vm_host.total_cpus
       vm_host.update(used_cores: spdk_cores)
-      hop_prep_reboot
+      hop_download_boot_image
     end
     donate
   end
 
+  label def download_boot_image
+    bud Prog::DownloadBootImage, {"image_name" => Config.default_boot_image_name, "version" => Config.default_boot_image_version}
```

Comment:
@fdr expressed concerns about automatically downloading images during development previously in the one of @furkansahin PRs.
https://github.com/ubicloud/ubicloud/pull/772#issuecomment-1780427420

Reply:
What do you think about https://github.com/ubicloud/ubicloud/pull/1597?